tests/scripts/minikube.sh

```diff
@@ -20,7 +20,10 @@ function wait_for_ssh() {
 function copy_image_to_cluster() {
     local build_image=$1
     local final_image=$2
-    docker save "${build_image}" | (eval "$(minikube docker-env --shell bash)" && docker load && docker tag "${build_image}" "${final_image}")
+    ${DOCKERCMD} save "${build_image}" | \
+        (eval "$(minikube docker-env --shell bash)" && \
```

Comment:
I don't think minikube fully supports podman, regretfully - see https://minikube.sigs.k8s.io/docs/drivers/podman/

Reply:
The Make changes are for setting Podman env variables on the host, it's not touching the driver used inside the Minikube VM. So effectively, this will allow a Podman client to build an image and push the image inside the registry of the Minikube instance.